src/main/scala/uk/gov/nationalarchives/fileformat/FileUtils.scala

```diff
@@ -10,6 +10,9 @@ import com.typesafe.config.ConfigFactory
 import graphql.codegen.GetOriginalPath.getOriginalPath.{Data, Variables, document}
 import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.GetObjectRequest
+import sttp.client.{HttpURLConnectionBackend, Identity, NothingT, Response, SttpBackend}
```

Comment:
Think: `HttpURLConnectionBackend` and `Response` are unnecessary imports

Reply:
This reminds me - what do people think about configuring the Jenkins build to spot unused imports?

I haven't tried it, but I think if you add `-Xlint:unused` (or possibly just `-Xlint) and `-Xfatal-warnings` then the build would fail. It would save us having to catch these in code review.